protocol/v2/ssv/validator/committee.go

```diff
@@ -231,13 +199,13 @@ func (c *Committee) ProcessMessage(logger *zap.Logger, msg *queue.SSVMessage) er
 		if err := qbftMsg.Validate(); err != nil {
 			return errors.Wrap(err, "invalid qbft Message")
 		}
-		c.mtx.Lock()
-		runner, exists := c.Runners[phase0.Slot(qbftMsg.Height)]
-		c.mtx.Unlock()
+		c.mtx.RLock()
+		r, exists := c.Runners[phase0.Slot(qbftMsg.Height)]
+		c.mtx.RUnlock()
 		if !exists {
-			return errors.New("no runner found for message's slot")
+			return errors.New(fmt.Sprintf("no runner found for consensus message's slot(qbtf height): %d", qbftMsg.Height))
```

Comment:
We always use slot as QBFT height, so we don't have to clarify this in logs
```suggestion
			return fmt.Errorf("no runner found for consensus message's slot %d", qbftMsg.Height)
```

Reply:
Applied in https://github.com/ssvlabs/ssv/pull/1784/commits/483c2ed41d9b789251d57e9d6ce4f3275e3f02c1